content/game_process.tex

```diff
@@ -6,15 +6,13 @@ \section{Game Process}
 \subsection{Structure of the Game}
 \label{sec:game_struct}
 
-A game consists of three parts, the first half, a half-time break, and the second half. Each half is \qty{10}{\minute} counted from the initial kick-off.
+A game consists of four parts, pregame meeting (see\,\ref*{sec:referee_team_communiation}), the first half, a half-time break, and the second half. Each half is \qty{10}{\minute} counted from the initial kick-off.
```

Comment:
pre-game, `\cref`.
Is there a reason to use `\,` here instead of `~`?

Reply:
> 

Yes, there where changes close by. So, I decided to do this. I do not know if it is the best way due to the dependencies.

Reply:
Done